types/enzyme/index.d.ts

```diff
@@ -364,6 +364,8 @@ export interface CommonWrapper<P = {}, S = {}, C = Component<P, S>> {
     length: number;
 }
 
+type Parameters<T> = T extends (...args: infer A) => any ? A : never
```

Comment:
This is built into TS version 3.1, so doesn't need to be declared inline _along with_ a bump to 3.1. We only need one or the other. The helper should be inlined like this only if we want to support a slightly older version of TS (2.9 or 3.0, I believe). Sorry for the confusion.

I'm fine with keeping the version at 3.1 and dropping this helper.